requirements.txt

```diff
@@ -4,7 +4,7 @@ certifi==2020.6.20
 chardet==3.0.4
 colorama==0.4.3; sys_platform == 'win32'
 distro==1.5.0
-docker==4.3.1
+git+git://github.com/docker/docker-py@refs/pull/2691/head#egg=docker
```

Comment:
I'm guessing this is a draft PR while we wait for the upstream change to be merged?